src/rate/nls/Rate.ts

```diff
@@ -0,0 +1,21 @@
+const messages = {
+	starLabels: `{denominator, plural, offset:0
```

Comment:
I don't follow why this widget needs `nls` at all. We should just be passing the decimal values to the radio group and returning them in the `onValue`.

Reply:
This is the recommendation by WAI and the way it's implemented in Material.

Reply:
This is the recommendation by WAI and the way it's implemented in Material.